extensions/arc/src/arc.ts

```diff
@@ -148,6 +148,23 @@ export async function makeNewWindow(options: MakeNewWindowOptions = {}): Promise
     end tell
   `);
 }
+export async function makeNewBlankWindow(): Promise<void> {
+  await runAppleScript(`
+    tell application "Arc"
+	    activate
+    end tell
+    delay(0.5)
```

Comment:
Why do we need this duplicate with delay?

Reply:
For some reason, it's Diane open a new windows unless you are ON that window, so I had to duplicate it to ensure the user is ON an arc window